File: src/main/java/uk/co/threebugs/mochiwhattotrade3/account/Account.java
package uk.co.threebugs.mochiwhattotrade3.account;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "account")
public class Account {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String name;
    private String metatraderAdvisorPath;

}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/account/Account.kt
package uk.co.threebugs.mochiwhattotrade3.account

import jakarta.persistence.*

@Entity
@Table(name = "account")
class Account (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int? = null,
    var name: String?,
    var metatraderAdvisorPath: String?
)

